package.json

```diff
@@ -4,11 +4,13 @@
   "description": "Make travel and accommodation easy and convenient.",
   "main": "index.js",
   "scripts": {
-    "start": "nodemon --exec babel-node src/index.js",
-    "test": "NODE_ENV=test npm run dropTables && NODE_ENV=test mocha --require @babel/polyfill --require @babel/register ./src/tests/*",
+    "dev-start": "nodemon --exec babel-node src/index.js",
+    "test": "NODE_ENV=test npm run createTables && NODE_ENV=test nyc --reporter=text --reporter=lcov mocha --timeout 1000  --require @babel/polyfill --require @babel/register src/**/**test.js",
+    "build": "babel src --out-dir dist && npm run createTables",
+    "start": "npm run build && node dist/index.js",
     "coverage": "nyc npm test && nyc report --reporter=text-lcov | coveralls",
```

Comment:
I don't understand why you changed the script because on the coverage it displays two tables which are wrong.
<img width="1236" alt="Screen Shot 2020-02-21 at 14 33 13" src="https://user-images.githubusercontent.com/50768983/75035163-89733a80-54b7-11ea-9d5f-f7a0fc22077b.png">